htmd/__init__.py

```diff
@@ -1,11 +1,17 @@
 from htmd.ui import *  # TODO: This will be deprecated in a few versions
-from htmd.config import config
```

Comment:
why are you removing config from __init__? It's actually necessary there. cannot be moved to ui